### ISSUE TITLE
[Frontend] Resolve accessibility violation in Real-time Balance Sync

### ISSUE THREAD

**Issue by emdevelopa** [emdevelopa/Stellar_Payment_API#1363]
Description
This task involves UX enhancement for the Real-time Balance Sync module.
The goal is to resolve accessibility violation in Real-time Balance Sync to improve the platform's styling, user interactions, and overall accessibility.

Requirements and context

- Must be secure, tested, and documented
- Adhere to the Drips Wave design and UI/UX standards
- Ensure compatibility across desktop and mobile browsers
- Specifically focused on frontend UX enhancement and responsiveness

Suggested execution

Fork the repo and create a branch
git checkout -b feature/fe-resolve-accessibility-violation-in-real-time-balance-sync
Implement changes

Review existing component in Real-time Balance Sync
Apply changes: Resolve accessibility violation in Real-time Balance Sync
Use clean CSS or tailwind variables for styling
Maintain state transitions and visual feedback
Test and commit

Test mobile responsiveness and interactive states
Check accessibility (a11y) using standard audits
Include screenshots or gifs in the PR
Example commit message
feat(frontend): resolve accessibility violation in Real-time Balance Sync

Guidelines

Complexity: Medium (150 points)
Assignment required before starting
PR description must include: Closes #[issue_id]

**Comment by adeboladee** [comment]
@adeboladee has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hello Maintainer, Please I would like to work on this issue


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/8659ae20-7370-49b4-878d-691f999bf977) or assign @adeboladee to this issue.

**Comment by khanavi272-spec** [comment]
@khanavi272-spec has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I would like to work on this issue 


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/8659ae20-7370-49b4-878d-691f999bf977) or assign @khanavi272-spec to this issue.

**Comment by davedumto** [comment]
@davedumto has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hello can i work on this issue please?


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/8659ae20-7370-49b4-878d-691f999bf977) or assign @davedumto to this issue.
